developer/guides/troubleshooting/tailwind-styles-being-overridden.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Tailwind styles are being overridden
+description: Resolve issues with Tailwind v4 or other @layer-based CSS frameworks being overridden by CSS resets
+---
+
+## Problem
+
+Starting from [version 4](https://tailwindcss.com/blog/tailwindcss-v4-alpha#designed-for-the-modern-web), Tailwind CSS uses `@layer` rules to define its styles.
+
+Makeswift by default includes a CSS reset from [normalize.css](https://github.com/necolas/normalize.css/).
+Because the CSS reset is unlayered, it'll override the Tailwind styles, even if the Tailwind styles are more specific.
+
+From [MDN @layer documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/@layer):
+
+![Image: CSS layer cascade](/images/layer-cascade.svg)
+
+
+## Solution
+
+Disable the built-in CSS reset by setting `enableCssReset={false}` on the `RootStyleRegistry`:
```

Comment:
Can we clarify where this snippet is? I had to go look at installation guide to realize it's in the `provider.tsx`. Maybe something like: 

"If you followed the installation docs, this would be located in your `provider.tsx`

Reply:
Sure. I just the files for Catalyst and non-Catalyst.